src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+
+const sortList = (array) => {
+  const newList = array.sort(
+    (a, b) =>
+      Number(b.getAttribute('data-salary').replace('$', '').replace(',', '')) -
+      Number(a.getAttribute('data-salary').replace('$', '').replace(',', '')),
```

Comment:
The `replace` method is used twice to remove characters from the salary string. Consider using a regular expression to simplify this operation, e.g., `replace(/[$,]/g, '')`.